no-computer.md

```diff
@@ -1,44 +1,69 @@
 # No computer project
 
+What I have chosen for my no-computer language is EMOTICONS.
+You may point out that emoticons ARE computer-based! But I argue that there is
+a meta-language, which can be used just as well in handwriting and in print,
+that computer-based emoticons try to mimic. That language, which I'm still
+calling emoticons, is essentially the idea of putting smileys and things in-line
+with other written text. So, that does not include full-scale drawings.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Emoticons allow people to more accurately describe their emotions over text-based communication.
+It is difficult without emoticons to concisely and accurately without nonverbal communication, 
+and arguably they are a better replacement
+for nonverbal communication than simply taking a long time to describe your feelings is.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Users are already in the context of communicating by writing. Emoticons as a DSL fit inside
+of that paradigm, so it is very easy for users to use, and easier to understand when
+first seeing them because they 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+It is interesting because it is decidedly NOT unambiguous.
```

Comment:
I'm really confused by this. Emoticons may be clearer than like facial expressions because you can go back to them, but often it can be super ambiguous what the typer means